docs/upgrading_to_okhttp_4.md

```diff
@@ -233,11 +233,47 @@ between ProGuard, OkHttp 4.x, and Kotlin-originated `.class` files. Make sure yo
 release if you’re using ProGuard,
 
 
+Gradle
+------
+
+OkHttp 4’s minimum requirements are Java 8+ and Android 5+. These requirements were
+[first introduced][require_android_5] with OkHttp 3.13.
+
+Here’s what you need in `build.gradle` to target Java 8 byte code for Kotlin, Java, and Android
+plugins respectively.
+
+```
+compileKotlin {
+  kotlinOptions {
+    jvmTarget = "1.8"
+  }
+}
+compileTestKotlin {
+  kotlinOptions {
+    jvmTarget = "1.8"
+  }
+}
+
+compileJava {
+  sourceCompatibility = 1.8
+  targetCompatibility = 1.8
```

Comment:
These two can use `JavaVersion` constants if you want

Reply:
Fixed.